### PR TITLE
Add current_user function

### PR DESCRIPTION
**Pull request by arbfranklin**
Potential implementation for #6687.

I wasn't sure how to best update the docs as this SQL-standard function doesn't easily fit into an existing category.

### CODE REVIEW

File: presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java [hunk 1]
```diff
@@ -7378,6 +7378,18 @@ public void testValuesWithUnusedColumns()
     }
 
     @Test
+    public void testCurrentUser()
+    {
+        MaterializedResult actual = computeActual("select current_user");
```
Comment (kokosing) [inline presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java]: why not to simply:
```
assertQuery("SELECT CURRENT_USER", "SELECT " + getSession.getUser());
```

ps. prefer keywords to be capitalized

File: presto-product-tests/src/main/resources/sql-tests/testcases/session_functions/currentUserAccessible.sql [hunk 2]
```diff
@@ -0,0 +1,2 @@
+-- database: presto; groups: qe, session_functions, functions
+select current_user
```
Comment (kokosing) [inline presto-product-tests/src/main/resources/sql-tests/testcases/session_functions/currentUserAccessible.sql]: I think it is already covered in `AbstractTestQueries`, no need to add it here.

**Comment by arbfranklin** [comment]
Thanks for the feedback @kokosing, your suggestions have been integrated.

**Comment by martint** [comment]
Superseded by https://github.com/prestodb/presto/pull/10108

**Pull request closed without merging**